manga-cli

```diff
@@ -70,8 +70,8 @@ select_chapter() {
 create_file() {
   if [ $file_format == "pdf" ]; then
     echo "Converting to pdf..."
-    convert $($ls -v $image_dir*.jpg) "$image_dir$name-$chapterNumber.pdf"
-    rm "$image_dir/*.jpg"
+    convert $(ls -v $image_dir*.jpg) "$image_dir$name-$chapterNumber.pdf"
```

Comment:
`$ls` allows to easily use aliases for `ls`

Reply:
It created issues on macOS because `ls` located in a different folder. If you want to keep it as a variable, no problems, I added an if statement above. Is it good now?